cishouseholds/pipeline/run.py

```diff
@@ -61,11 +62,15 @@ def run_from_config():
     with spark_description_set("adding run status"):
         add_run_status(run_id, "started")
     pipeline_error_count = None
+
     try:
+        config_copy = copy.deepcopy(config)
```

Comment:
There shouldn't be a need to take a deepcopy of the config, as the validation logic shouldn't modify the config object